src/components/operations/operation-details/ko/runtime/operation-details.html

```diff
@@ -175,12 +175,33 @@ <h4>Request body</h4>
                     <!-- /ko -->
 
                     <!-- ko if: representation.examples -->
+                    <!-- ko if: representation.examples.length > 1 -->
+                        <div class="row flex flex-row example-form-control">
+                            <div class="col-12">
+                                <label data-bind="attr : { for: 'select' + representation.contentType + '_' + $index() }, text: 'Examples'"></label>
+                            </div>
+                            <div class="col-4">
+                                <div class="form-group">
+                                    <select class="form-control" data-bind="options: representation.examples, value: $component.selectedRepresentatnionsValue()[representation.contentType], event: {change: $component.permissionChanged} ,optionsValue: 'title', optionsText: 'title', attr : {title: 'Represantation examples selector', id: 'select-' + representation.contentType + '_' + $index() }"></select>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- ko foreach: { data: representation.examples, as: 'example' } -->
+                        <code-snippet data-bind="visible: $component.selectedRepresentatnionsValue()[representation.contentType]() == example.title"
+                            params="{ title: example.title, content: example.value, language: example.format, description: example.description }">
+                        </code-snippet>
+                        <!-- /ko -->
+                    <!-- /ko -->
+
+                    <!-- ko if: representation.examples.length == 1 -->
```

Comment:
since `representation.examples.length == 1`, why do we foreach `representation.examples`? I think it would make more sense to just use `representation.examples[0]`